DuckDuckGo/TabManager.swift

```diff
@@ -105,10 +121,11 @@ class TabManager {
         let controller = buildController(forTab: tab, url: url)
         tabControllerCache.append(controller)
 
-        if let index = model.currentIndex, inBackground {
+        let index = model.currentIndex
+        if inBackground {
             model.insert(tab: tab, at: index + 1)
         } else {
-            model.add(tab: tab)
+            model.insert(tab: tab, at: model.count - 1)
```

Comment:
model.last computed property would be more readable and safer to have here. But not necessary :)